docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM fpco/stack-build:lts-9.10
```

Comment:
@vsoch this could be easily simplified by switching to newer LTS e.g. https://github.com/qrilka/penrose/commit/2faf1ec4736c5dffcdfcba41191466a12ab2da6a builds fine for me
With that change you could use `fpco/stack-build:lts-11.22` and don't spend time on installing GHC, alex and happy and `--system-ghc` won't be needed (BTW there's `stack setup` to get proper GHC with Stack)
Hope this helps

Reply:
Ah this is so helpful! Let me give this a try locally, and I'll update the PR. I really appreciate your testing this out and advice on this!